docs/worker-node/install-cvmfs.md

```diff
@@ -84,7 +84,9 @@ CVMFS uses automount, and the steps to configure it are different on EL6 vs EL7.
         :::console
         root@host # service autofs restart
 
-### For EL7 hosts
+
+<a name="for-el7-hosts"></a>
```

Comment:
Why are we keeping the old anchor?

Reply:
In case we have links from external places?

Reply:
Eh, I'd rather just get rid of it. An external link with an anchor that doesn't exist anymore will still bring them to the same page